### ISSUE TITLE
GNOME 43 runtime EOL

### ISSUE THREAD

**Issue by PresGas** [flathub/org.apache.directory.studio#2]
Placeholder issue. Flatpak will warn about this on updates.
```
Looking for updates…
Info: org.gnome.Platform//43 is end-of-life, with reason:
   The GNOME 43 runtime is no longer supported as of September 20, 2023. Please ask your application developer to migrate to a supported platform.
Applications using this runtime:
   org.apache.directory.studio
```

**Comment by wjt** [comment]
Addressed by #3 (though one might consider updating to 45 I suppose).

**Comment by wjt** [comment]
Did that in #4
